specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkManagerActiveConfiguration.json

```diff
@@ -0,0 +1,240 @@
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/activeConfiguration/{locationName}": {
+      "parameters": [
+        {
+          "$ref": "./network.json#/parameters/ApiVersionParameter"
+        },
+        {
+          "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+        },
+        {
+          "$ref": "#/parameters/ResourceGroupNameParameter"
+        },
+        {
+          "$ref": "#/parameters/NetworkManagerNameParameter"
+        },
+        {
+          "name": "locationName",
+          "in": "path",
+          "required": true,
+          "type": "string",
+          "description": "The location Name."
+        }
+      ],
+      "get": {
+        "tags": [
+          "activeConfiguration"
+        ],
+        "operationId": "ActiveConfigurations_Get",
+        "description": "The Get active configuration operation to get an active configuration",
+        "responses": {
+          "200": {
+            "description": "Get successful. The operation returns the specified Managed Network active configuration resource.",
+            "schema": {
+              "$ref": "#/definitions/ActiveConfiguration"
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ConnectivityConfigurationsGet": {
+            "$ref": "./examples/NetworkManagerActiveConfigurationGet.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ActiveConfiguration": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ActiveConfigurationProperties",
+          "description": "The Active Configuration properties"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "Configuration and location lists to commit."
+    },
+    "ActiveConfigurationProperties": {
+      "properties": {
+        "commits": {
```

Comment:
>commits [](start = 9, length = 7)

this is confusing because, this is not the schema of 'commit' resource and this api is about getting the active configuration .. not about the commits. let me know

Reply:
This commit is different from the resource commit that we difined in network commits. it is a list of commitem, which commititem is combination of configuration, group, rules and deployment time